src/_scss/pages/homepage/features/_covidFeature.scss

```
@@ -0,0 +1,238 @@
+.feature-covid {
+    
+    @include media ($medium-screen) {
+        margin-top: -5%;
+    }
+
+    //ROW 1: OFFICIAL SPENDING DATA
+    .official-spending-data__content-wrapper {
+
+        @include media($medium-screen) {
+            @include display(flex);
+            @include justify-content(center);
+        }
+
+        .official-spending-data__text {
+            display: block;
+            margin: rem(-50) rem(10) rem(10) rem(10);
+
+            @include media($medium-screen) {
+                max-width: 40%;
+                margin: rem(0) rem(85) rem(60) rem(0);
+            }
+        }
+
+        .feature-covid-official-spending-data__image-wrapper {
+            @include display(flex);
+            @include justify-content(center);
+
+            .feature-covid-official-spending-data__image-mobile {
+                display: block;
+                max-width: 75%;
+                margin: rem(20) rem(0) rem(20) rem(0);
+    
+                @include media ($medium-screen) {
+                    display: none;
+                }
+            }
+
+            .feature-covid-official-spending-data__image {
+                display: none;
+    
+                @include media ($medium-screen) {
+                    display: block;
+                    width: 35vw;
+                    height: 25vw;
+                    max-width: none;
+                }
+            }
+        }
+    }
+
+    //ROW 2: ADVANCED SEARCH FILTER + SPENDING PROFILE 
+    .advanced-search-and-spending-profile__wrapper {
+        display: block;
+        margin-top: rem(60);
+
+        @include media ($medium-screen) {
+            margin-top: rem(10);
+            margin-left: rem(103);
+            @include display(flex);
+            @include justify-content(center);
+
+        }
+
+        .advanced-search__content-wrapper {
+            margin-top: rem(100);
+
+            @include media ($medium-screen) {
+                max-width: 45%;
+                @include display(flex);
+                @include flex-direction(column);
+            }
+        }
+
+        .advanced-search__content-wrapper-right {
+            margin-top: rem(150);
+            margin-bottom: rem(100);
+
+            @include media ($medium-screen) {
+                max-width: 45%;
```

Max Width is not gonna work in IE. Lets must make this width

```suggestion
                width: 45%;
```